### ISSUE TITLE
Module Not Found for 'lyft.availability'

### ISSUE THREAD

**Issue by pmdbt** [vmanikes/Lyft-Python-SDK#9]
I am having another issue with importing the module. I took a look at the way the code is laid out on github and it seems that lyft.availability should be the correct way to access the class Availability. However, I'm receiving an error that the module isn't available. Here is my code and error output. 

```python

from lyft.authentication.auth import LyftPublicAuth
from lyft.availability import Availability

#constants for API authentication
CLIENT_ID = '***'
CLIENT_SECRET = '***'
CLIENT_TOKEN = '***'

current_token = LyftPublicAuth({"client_id": CLIENT_ID, "client_secret": CLIENT_SECRET}, sandbox_mode = False).get_access_token()

#testing
print(current_token["access_token"])
print(current_token["token_type"])
availability_obj = Availability(current_token["token_type"], current_token["access_token"])

response = availability_obj.get_ride_types(-118.25009, 33.97908, 'lyft')
print(response)
```

The output is: 

`
ModuleNotFoundError: No module named 'lyft.availability'
`

Any ideas on a fix?

**Comment by vmanikes** [comment]
Yes, I am actually working on pushing an update on that in Python repositories, in the meantime you can download the `availability.py` code from here`https://github.com/vmanikes/Lyft-Python-SDK/tree/master/lyft` and put in in your local Lyft-Python-SDK installation. For example, I am using Venv, so you can put the code in `<your_project_name/venv/lib64/python3.6/site-packages/lyft/availability.py`. You should receive a update for this repo by the weekend

**Comment by pmdbt** [comment]
Ok, I will check back again early next week! Thanks for keeping on top of this!

**Comment by houseowl** [comment]
Hi @vmanikes, have you pushed a new update in Python repositories yet?

**Comment by vmanikes** [comment]
Unfortunately, I wasn't able to upload it to python repositories, Instead, I have uploaded the dist into the repo and updated the readme to reflect the installation instructions.
